flocker/node/agents/testtools/_blockdevice.py

```diff
@@ -891,117 +891,81 @@ class InvalidConfig(Exception):
     """
 
 
-# Highly duplicative of other constants.  FLOC-2584.
-class ProviderType(Names):
-    """
-    Kinds of compute/storage cloud providers for which this module is able to
-    build ``IBlockDeviceAPI`` providers.
-    """
-    openstack = NamedConstant()
-    aws = NamedConstant()
-    gce = NamedConstant()
-    rackspace = NamedConstant()
+# XXX Remember to comment on or close FLOC-2584.
```

Comment:
Remember to comment on or close FLOC-2584.

Honestly... I'd just close it. This is the sort of issue I don't fully understand why we create. It's not like there's ever a time where we just choose to take a random issue off the backlog, and what needs to be done to clean it up is pretty obvious to someone looking through the code. I'm not sure that the issue provides enough relevant content above what the code was to outweigh the fact that it could become stale while the code changes.